test/services/HolderOfAuthenticationCodeSpec.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import uk.gov.hmrc.play.test.UnitSpec
+
+class HolderOfAuthenticationCodeSpec extends UnitSpec {
+
+  "Holder of Authentication Code" should {
+    "read from file" in {
+      val codes = HolderOfAuthenticationCode.all
+      codes should contain(HolderOfAuthenticationCode("ACE")) // First in file
```

Comment:
Remind me to show you the scala test asserts for contains inOrder.... it's pretty cool and a bit stronger than this